src/app/components/organizacion/organizacion-prestaciones.component.ts

```diff
@@ -0,0 +1,160 @@
+        });
+    }
+
+    /*
+     * AGREGAR
+     */
+    onAgregar() {
+        this.agregando = true;
+        this.cancelarEditar();
+    }
+
+    cancelarAgregar() {
+        this.agregando = false;
+        this.modelo = {
+            prestacion: null,
+            descripcion: null,
+        };
+    }
+
+    agregarOferta() {
+        if (this.modelo.prestacion) {
+            this.organizacion.ofertaPrestacional.push(this.modelo);
+            this.organizacionService.save(this.organizacion).subscribe(organizacion => {
+                this.plex.info('info', 'La oferta prestacional fue agregada');
+                this.ofertaPrestacionalSeleccionada = null;
+                this.cancelarAgregar();
+                this.cancelarEditar();
+            });
+        }
+    }
+
+    /*
+     * EDITAR
+     */
+    onEditar(oferta: { _id: string, prestacion: ITipoPrestacion, detalle: string }) {
+        this.ofertaPrestacionalSeleccionada = oferta;
+        this.prestacion = oferta.prestacion;
+        this.detalle = oferta.detalle;
+        this.editando = true;
+    }
+
+    cancelarEditar() {
+        this.ofertaPrestacionalSeleccionada = null;
+        this.editando = false;
+        this.prestacion = null;
+        this.detalle = null;
+    }
+
+    editarOferta() {
+        this.ofertaPrestacionalSeleccionada.prestacion = this.prestacion;
+        this.ofertaPrestacionalSeleccionada.detalle = this.detalle;
+
+        let i = 0;
+        for (const oferta of this.organizacion.ofertaPrestacional) {
+            if (oferta._id === this.ofertaPrestacionalSeleccionada._id) {
+                this.organizacion.ofertaPrestacional[i] = this.ofertaPrestacionalSeleccionada;
+            }
+            i++;
+        }
+
+        this.organizacionService.save(this.organizacion).subscribe(organizacion => {
+            this.plex.info('info', 'La oferta prestacional fue editada');
+            this.cancelarEditar();
+        });
+    }
+
+    hayCambios() {
+        return ((this.ofertaPrestacionalSeleccionada.prestacion.fsn !== this.prestacion.fsn) || (this.ofertaPrestacionalSeleccionada.detalle !== this.detalle));
+    }
+
+    /*
+     * ELIMINAR
+     */
+    eliminarOferta(ofertaSeleccionada: { _id: string, prestacion: ITipoPrestacion, detalle: string }) {
+        this.ofertaPrestacionalSeleccionada = ofertaSeleccionada;
+        this.plex.confirm('Eliminar oferta prestacional "' + this.ofertaPrestacionalSeleccionada.prestacion.fsn + '"', '¿Eliminar oferta prestacional?').then(confirmacion => {
+            if (confirmacion) {
+                let i = 0;
+                let index = 0;
+                for (const oferta of this.organizacion.ofertaPrestacional) {
+                    if (oferta._id === this.ofertaPrestacionalSeleccionada._id) {
+                        index = i;
+                    }
+                    i++;
+                }
+                this.organizacion.ofertaPrestacional.splice(index, 1);
```

Comment:
Porqué no usar el método `filter` en lugar de interar el array?
```suggestion
this.organizacion.ofertaPrestacional = this.organizacion.ofertaPrestacional.filter(e => e._id !== this.ofertaPrestacionalSeleccionada._id);
```